backend/de.metas.adempiere.adempiere/migration/src/main/sql/postgresql/system/10-de.metas.adempiere/5683050_sys_gh15007_CostDetail_SourceAmtAndCurrency.sql

```diff
@@ -0,0 +1,117 @@
+;
+
+-- 2023-03-28T15:56:07.787Z
+INSERT INTO AD_Column_Trl (AD_Language,AD_Column_ID, Name, IsTranslated,AD_Client_ID,AD_Org_ID,Created,Createdby,Updated,UpdatedBy,IsActive) SELECT l.AD_Language, t.AD_Column_ID, t.Name, 'N',t.AD_Client_ID,t.AD_Org_ID,t.Created,t.Createdby,t.Updated,t.UpdatedBy,'Y' FROM AD_Language l, AD_Column t WHERE l.IsActive='Y'AND (l.IsSystemLanguage='Y' OR l.IsBaseLanguage='Y') AND t.AD_Column_ID=586370 AND NOT EXISTS (SELECT 1 FROM AD_Column_Trl tt WHERE tt.AD_Language=l.AD_Language AND tt.AD_Column_ID=t.AD_Column_ID)
+;
+
+-- 2023-03-28T15:56:08.375Z
+/* DDL */  select update_Column_Translation_From_AD_Element(578756) 
+;
+
+-- 2023-03-28T15:56:10.407Z
+/* DDL */ SELECT public.db_alter_table('M_CostDetail','ALTER TABLE public.M_CostDetail ADD COLUMN Source_Currency_ID NUMERIC(10) NOT NULL')
+;
+
+-- 2023-03-28T15:56:10.545Z
+ALTER TABLE M_CostDetail ADD CONSTRAINT SourceCurrency_MCostDetail FOREIGN KEY (Source_Currency_ID) REFERENCES public.C_Currency DEFERRABLE INITIALLY DEFERRED
+;
+
+-- 2023-03-28T15:56:37.972Z
+INSERT INTO t_alter_column values('m_costdetail','Source_Currency_ID','NUMERIC(10)',null,null)
+;
+
+-- Column: M_CostDetail.Source_Currency_ID
+-- 2023-03-28T16:02:23.661Z
+UPDATE AD_Column SET IsMandatory='N',Updated=TO_TIMESTAMP('2023-03-28 19:02:23','YYYY-MM-DD HH24:MI:SS'),UpdatedBy=100 WHERE AD_Column_ID=586370
+;
+
+-- 2023-03-28T16:02:26.417Z
+INSERT INTO t_alter_column values('m_costdetail','Source_Currency_ID','NUMERIC(10)',null,null)
+;
+
+-- 2023-03-28T16:02:26.422Z
+INSERT INTO t_alter_column values('m_costdetail','Source_Currency_ID',null,'NULL',null)
+;
+
+-- Column: M_CostDetail.SourceAmt
+-- 2023-03-28T16:02:36.031Z
+UPDATE AD_Column SET IsMandatory='N',Updated=TO_TIMESTAMP('2023-03-28 19:02:36','YYYY-MM-DD HH24:MI:SS'),UpdatedBy=100 WHERE AD_Column_ID=586369
+;
+
+-- 2023-03-28T16:02:39.091Z
+INSERT INTO t_alter_column values('m_costdetail','SourceAmt','NUMERIC',null,null)
+;
+
+-- 2023-03-28T16:02:39.097Z
+INSERT INTO t_alter_column values('m_costdetail','SourceAmt',null,'NULL',null)
+;
+
```

Comment:
pls display those 2 fields in the window. 
IMHO: No need to display in grid mode.